src/client/utils/funcs/fieldValueToString.ts

```diff
@@ -1,17 +1,35 @@
-import type { Field } from "../../../shared/manifest";
+import type { Field, ManifestRelation } from "../../../shared/manifest";
 
-export const fieldValueToString = (field: Field, value: any) => {
-  if (field.type === "date") {
-    value = `${value.day}/${value.month}/${value.year}`;
+export const fieldValueToString = (
+  field: Field | ManifestRelation,
+  value: any
+) => {
+  if (!field || !value) return value;
+  switch (field.type) {
+    case "relatesToOne":
```

Comment:
Please extract these constants to an enum just to be tidy

Reply:
Done!